src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String LONG_LINE = "____________________________________________________________";
+    public static final int MAX_ARRAY_SIZE = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreetings();
+        Task[] taskList = new Task[100];
+        String line = "Start";
+        Scanner in = new Scanner(System.in);
+        int listCount = 0;
+        while (!line.equals("bye")) {
+            line = in.nextLine();
+            if (!line.equals("bye")) {
+                listCount = handleUserInputs(line, taskList, listCount);
+            }
+            System.out.println(LONG_LINE);
+        }
+        printExit();
+        return;
+    }
+
+
+    private static void printList(Task[] s, int listCount) {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < listCount; ++i) {
+            String index = Integer.toString(i + 1);
+            System.out.println(index + '.' + s[i].toString());
+        }
+    }
+
+    private static void printAddedTaskCommand(Task[] taskList, int listCount) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + taskList[listCount].toString());
+        System.out.println("Now you have " + Integer.toString(listCount + 1) + " tasks in the list.");
+    }
+
+    private static void printGreetings() {
+        System.out.println(LONG_LINE);
+        System.out.println("Hello I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(LONG_LINE);
+    }
+
+    private static void printExit() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(LONG_LINE);
+    }
+
```

Comment:
I like how you abstracted the print methods into separate methods. Makes the code look cleaner